### PR TITLE
Added usage example for fill_quad

### PR DESCRIPTION
**Pull request by SimonRhook**
# Overview

Added new usage example for fill_quad including C\# OOP and Top level, C++ and Python

# Files Included

Splashkit Function: `fill_quad`
- [x] Title and explanation (.txt)
- [x] C++ code
- [x] C# code (top-level statements)
- [x] C# code (Object-Oriented Programming)
- [x] Python code
- [x] Screenshot

# Usage Example Checks

- [x] Simple, clear demonstration of the function
- [x] Code uses Splashkit functions
- [x] Tested in Chrome and Firefox

### CODE REVIEW

**Review by NoahJCross — APPROVED** [review]
# Peer Review
I've reviewed the fill_quad usage example which draws a diamond with 4 quads. The code runs with no errors.

# Checks
- [x] All required files are present.
  - [x] Title and explanation (.txt)
  - [x] C++ code
  - [x] C# code (top-level statements)
  - [x] C# code (Object-Oriented Programming)
  - [x] Python code
- [x] Code correctly uses Splashkit functions.
- [x] Code clearly demonstrates the function.
- [x] All versions maintain the same structure and comments.

# Tests done
- [x] C++ code ran correctly.
- [x] C# top level code ran correctly.
- [x] C# OOP code ran correctly.
- [x] Python code ran correctly.

File: public/usage-examples/graphics/fill_quad/fill_quad-1-simple-oop.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
+using SplashKitSDK;
+
+namespace WriteLineInt
```
Comment (breezy-codes) [inline public/usage-examples/graphics/fill_quad/fill_quad-1-simple-oop.cs:3]: The namespace needs to be updated.

**Review by breezy-codes — CHANGES_REQUESTED** [review]
I've reviewed this PR, all code works correctly. Just one small change requested before I can approve this PR.

**Review by breezy-codes — APPROVED** [review]
I've reviewed this pr, all code runs correctly and is free of errors. Code comments and structure remain consistent over all files.

I approve this PR

- [X] All required files are present.
  - [X] Title and explanation (.txt)
  - [X] C++ code
  - [X] C# code (top-level statements)
  - [X] C# code (Object-Oriented Programming)
  - [X] Python code
- [X] Code correctly uses Splashkit functions.
- [X] Code clearly demonstrates the function.
- [X] All versions maintain the same structure and comments.
- [X] C++ code ran correctly.
- [X] C# top level code ran correctly.
- [X] C# OOP code ran correctly.
- [X] Python code ran correctly.

**Comment by omckeon** [comment]
See my review comments on the "Create usage example for draw_line_on_window" PR [here](https://github.com/thoth-tech/splashkit.io-starlight/pull/235#pullrequestreview-2472288685) and make relevant updates.

**Review by omckeon — APPROVED** [review]

**Pull request merged**